### ISSUE TITLE
Yaapu LUA error Ethos 1.5.11,  Tandem X20s

### ISSUE THREAD

**Issue by fdapport** [FrSkyRC/ETHOS-Feedback-Community#4371]
TX: Tandem X20s
OS TX: Ethos 1.5.11 FLEX
RX : R9 Slim +, LBT version 
OS Rx: 1.3.1
LUA: Yaapu version 1.1.0
Controller: Matek F405 WIN
SO Controller: Arduplane V.4.0.8

On Open TX all work, include receiving telemetry data. I have installed Yaapu LUA end I have the following Error:
ONLY GPS coordinate sensors is discovered: See below

![20240811_181831](https://github.com/user-attachments/assets/a153ea1a-ff56-4413-b73c-5999d002be18)

If I run the Yaapu LUA Telemetry Debug all the sensors are received:

![20240811_181858](https://github.com/user-attachments/assets/ae826a2f-b93d-4e8a-b3ce-494b90c2fe68)

If I set a LUA Sensors in the model

![20240811_182017](https://github.com/user-attachments/assets/cd8bf951-b7d3-4998-8b77-6ec125ba7300)

I have the following error:

![20240811_220249](https://github.com/user-attachments/assets/dfea8764-a3a1-4f11-90a4-6708bb22cfe2)

The error is this but I am not able to fix it. Could somebody can help. Thanks,

![LUA - Copia](https://github.com/user-attachments/assets/2772cacd-20d7-4a63-8269-cc10543dd648)

Thanks, Franco

**Comment by mawzthefinn** [comment]
You should report this to Yaapu, it's an issue in his script, not in ETHOS>

**Comment by mgsanna** [comment]
Buondì Franco devi aggiornare il widget Yappu credo ci sia già la versione ottimizzata e nuovaInviato da iPhoneIl giorno 12 ago 2024, alle ore 15:01, fdapport ***@***.***> ha scritto:﻿
TX: Tandem X20s
OS TX: Ethos 1.5.11 FLEX
RX : R9 Slim +, LBT version
OS Rx: 1.3.1
LUA: Yaapu version 1.1.0
Controller: Matek F405 WIN
SO Controller: Arduplane V.4.0.8
On Open TX all work, include receiving telemetry data. I have installed Yaapu LUA end I have the following Error:
ONLY GPS coordinate sensors is discovered: See below
20240811_181831.jpg (view on web)
If I run the Yaapu LUA Telemetry Debug all the sensors are received:
20240811_181858.jpg (view on web)
If I set a LUA Sensors in the model
20240811_182017.jpg (view on web)
I have the following error:
20240811_220249.jpg (view on web)
The error is this but I am not able to fix it. Could somebody can help. Thanks,
LUA.-.Copia.jpg (view on web)
Thanks, Franco
